python/ray/serve/tests/test_standalone.py

```diff
@@ -255,6 +275,9 @@ def third_actor_removed():
     wait_for_condition(third_actor_removed)
     ray.get(block_until_http_ready.remote("http://127.0.0.1:8005/-/routes"))
 
+    # TODO: remove before merge
+    assert False, "intentionally failing this to check linux CI"
```

Comment:
What were you checking in the CI?

Reply:
I'm checking that this tests get called in both macOS and Linux. Not just one of them.